packages/web-pocs/src/components/CollapsibleOcTable/CollapsibleOcTable.vue

```diff
@@ -0,0 +1,757 @@
+            @vue:mounted="$emit(constants.EVENT_TROW_MOUNTED, item, $refs[`row-${trIndex}`][0])"
+            @dragstart="dragStart(item, $event)"
+            @drop="dropRowEvent(itemDomSelector(item), $event)"
+            @dragenter.prevent="dropRowStyling(itemDomSelector(item), false, $event)"
+            @dragleave.prevent="dropRowStyling(itemDomSelector(item), true, $event)"
+            @mouseleave="dropRowStyling(itemDomSelector(item), true, $event)"
+            @dragover="dragOver($event)"
+          >
+            <oc-td
+              v-for="(field, tdIndex) in fields"
+              :key="'oc-tbody-td-' + cellKey(field, tdIndex, item)"
+              v-bind="extractTdProps(field, tdIndex, item)"
+            >
+              <slot v-if="isFieldTypeSlot(field)" :name="field.name" :item="item" />
+              <template v-else-if="isFieldTypeCallback(field)">
+                {{ field.callback(item[field.name]) }}
+              </template>
+              <template v-else>{{ item[field.name] }}</template>
+            </oc-td>
+          </oc-tr>
+        </template>
+      </oc-tbody>
+      <tfoot v-if="$slots.footer" class="oc-table-footer">
+        <tr class="oc-table-footer-row">
+          <td :colspan="fullColspan" class="oc-table-footer-cell">
+            <!-- @slot Footer of the table -->
+            <slot name="footer" />
+          </td>
+        </tr>
+      </tfoot>
+      <Teleport v-if="dragItem" to="body">
+        <oc-ghost-element
+          ref="ghostElement"
+          :preview-items="[dragItem, ...dragSelection]"
+        ></oc-ghost-element>
+      </Teleport>
+    </table>
+  </div>
+</template>
+<script lang="ts">
+// import OcThead from '../../../../design-system/src/components/_OcTableHeader/_OcTableHeader.vue'
+// import OcTbody from '../../../../design-system/src/components/_OcTableBody/_OcTableBody.vue'
+// import OcTr from '../../../../design-system/src/components/_OcTableRow/_OcTableRow.vue'
+// import OcTh from '../../../../design-system/src/components/_OcTableCellHead/_OcTableCellHead.vue'
+// import OcTd from '../../../../design-system/src/components/_OcTableCellData/_OcTableCellData.vue'
+// import OcGhostElement from '../../../../design-system/src/components/_OcGhostElement/_OcGhostElement.vue'
+// import OcButton from '../../../../design-system/src/components/OcButton/OcButton.vue'
+import SortMixin from '../../../../design-system/src/mixins/sort'
+import { getSizeClass } from '../../../../design-system/src/utils/sizeClasses'
+import { defineComponent, ref } from 'vue'
+
+import {
+  EVENT_THEAD_CLICKED,
+  EVENT_TROW_CLICKED,
+  EVENT_TROW_MOUNTED,
+  EVENT_TROW_CONTEXTMENU,
+  EVENT_ITEM_DROPPED,
+  EVENT_ITEM_DRAGGED
+} from '../../../../design-system/src/helpers/constants'
+
+/**
+ * A table component with dynamic layout and data.
+ */
+export default defineComponent({
+  name: 'OcTable',
+  status: 'ready',
+  release: '2.1.0',
+  components: {
+    // OcThead,
+    // OcTbody,
```

Comment:
Could be deleted also? They should be available through the runtime where the ODS is registered